### PR TITLE
when vive is on i5 report a correct target frame rate so we can see that in metaverse stats

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 23b74c7aff35ca55c6fc782c9bd557c47b3f61b1 available at: https://gist.github.com/18f849979485b8b809a294e9780522c4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-23b74c7.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-23b74c7.dmg

**Comment by ZappoMan** [comment]
researching why the new steam beta acts a little different

**Comment by hifi-gustavo** [comment]
Build console output for commit: bfe6ebb44fe69a5eed5699046d87cd8ff4e9124d available at: https://gist.github.com/18f849979485b8b809a294e9780522c4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-bfe6ebb.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-bfe6ebb.dmg

**Comment by ZappoMan** [comment]
tested and it's now working.

File: plugins/openvr/src/OpenVrDisplayPlugin.h [hunk 1]
```diff
@@ -82,4 +83,6 @@ class OpenVrDisplayPlugin : public HmdDisplayPlugin {
     std::shared_ptr<OpenVrSubmitThread> _submitThread;
     std::shared_ptr<gl::OffscreenContext> _submitCanvas;
     friend class OpenVrSubmitThread;
+
+    bool _asyncReprojectionActive { false };
```
Comment (jherico) [inline plugins/openvr/src/OpenVrDisplayPlugin.h]: I suggest we move this variable to `HmdDisplayPlugin`, the base class for both Oculus and OpenVR.  Future HMDs will either support this or not, so we should make sure it's baked into the base class for HMDs.  Also, you should update the Oculus plugin to set the value to true always.

Comment (jherico) [inline plugins/openvr/src/OpenVrDisplayPlugin.h]: Yes, it's always true for Oculus.

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp [hunk 2]
```diff
@@ -354,6 +353,13 @@ bool OpenVrDisplayPlugin::isSupported() const {
     return openVrSupported();
 }
 
+float OpenVrDisplayPlugin::getTargetFrameRate() const {
+    if (forceInterleavedReprojection && !_asyncReprojectionActive) {
+        return TARGET_RATE_OpenVr / 2.0f;
+    }
+    return TARGET_RATE_OpenVr;
```
Comment (jherico) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: we should eliminate this hardcoded value and replace it with a call to `vr::VRSettings()->getFloat()` with the appropriate parameters.

File: plugins/openvr/src/OpenVrDisplayPlugin.h (hunk 1 above)
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.h]: I have updated the oculus plugin to report true

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp (hunk 2 above)
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: would you accept that in a different PR?

Comment (jherico) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: yes

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d94df346fe60d6071fe7a8491aef7a40ac546bb available at: https://gist.github.com/18f849979485b8b809a294e9780522c4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-6d94df3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8922-6d94df3.dmg

**Pull request merged**